lib/tarjan_tree.rb

```diff
@@ -52,10 +55,15 @@ def create_tree(ontology)
   end
 
   def create_groups(ontology)
+    symbols = self.symbols - subclass_sym
     strongly_connected_components.each do |symbol_ids|
-      symbols = OntologyMember::Symbol.find(symbol_ids)
-      name = group_name_for(symbols)
-      SymbolGroup.create!(ontology: ontology, symbols: symbols, name: name)
+      syms = OntologyMember::Symbol.find(symbol_ids)
```

Comment:
Please give a more suitable name to the variable `syms`. How about `strongly_connected_symbols` or `equivalent_symbols` or `something_else_symbols`?